doc/classes/PopupMenu.xml

```diff
@@ -13,6 +13,15 @@
 	<tutorials>
 	</tutorials>
 	<methods>
+		<method name="activate_item_by_event">
+			<return type="bool" />
+			<param index="0" name="event" type="InputEvent" />
+			<param index="1" name="for_global_only" type="bool" default="false" />
+			<description>
+				Checks the supplied [InputEvent] ([param event]) against the [PopupMenu]'s shortcuts and accelerators, and activates any items with matching events. Returns true if an item was successfully activated.
+				[b]Note:[/b] Certain [Control]s such as [MenuButton] handle this method automatically.
```

Comment:
```suggestion
				Checks the supplied [param event] against the [PopupMenu]'s shortcuts and accelerators, and activates any items with matching events. Returns [code]true[/code] if an item was successfully activated.
				[b]Note:[/b] Certain [Control]s, such as [MenuButton], will call this method automatically.
```

Reply:
I think you should also mention what does `for_global_only` do.

Reply:
Also the claim that it activates *any* items is wrong, it activates only the first one:
```suggestion
				Checks the provided [InputEvent] ([param event]) against the [PopupMenu]'s shortcuts and accelerators, and activates the first item with matching events. Returns [code]true[/code] if an item was successfully activated.
				[b]Note:[/b] Certain [Control]s, such as [MenuButton], will call this method automatically.
```

Reply:
Fixed.